prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -203,90 +204,103 @@
-                                 title="Site Name"
-                                 [makeBold]="true">
-            {{ site.siteName | default }}
-          </app-enrollee-property>
-
-          <app-enrollee-property *ngIf="site.careSettingCode === CareSettingEnum.HEALTH_AUTHORITY"
-                                 title="Facility Name"
-                                 [makeBold]="true">
-            {{ site.facilityName | default }}
-          </app-enrollee-property>
-
-          <app-enrollee-property *ngIf="site.careSettingCode !== CareSettingEnum.HEALTH_AUTHORITY"
-                                 title="Site ID"
-                                 [makeBold]="true">
-            {{ site.pec | default }}
-          </app-enrollee-property>
-
-          <app-enrollee-property title="Job Title"
-                                 [makeBold]="true">
-            {{ site.jobTitle | default }}
-          </app-enrollee-property>
-
-          <app-enrollee-property title="Site Address"
-                                 [makeBold]="true">
-            <!-- <div class="mb-3">{{ site.physicalAddress?.street | default }}</div> -->
-            <app-enrollee-property title="Street">
-              {{ site.physicalAddress?.street | default }}
+            <app-enrollee-property *ngIf="site.careSettingCode !== CareSettingEnum.HEALTH_AUTHORITY"
+                                   title="Site Name"
+                                   [makeBold]="true">
+              {{ site.siteName | default }}
             </app-enrollee-property>
 
-            <app-enrollee-property title="City">
-              {{ site.physicalAddress?.city | default }}
+            <app-enrollee-property *ngIf="site.careSettingCode === CareSettingEnum.HEALTH_AUTHORITY"
+                                   title="Facility Name"
+                                   [makeBold]="true">
+              {{ site.facilityName | default }}
             </app-enrollee-property>
 
-            <app-enrollee-property title="Province">
-              {{ site.physicalAddress?.provinceCode | configCode: 'provinces' | default }}
+            <app-enrollee-property *ngIf="site.careSettingCode !== CareSettingEnum.HEALTH_AUTHORITY"
+                                   title="Site ID"
+                                   [makeBold]="true">
+              {{ site.pec | default }}
             </app-enrollee-property>
 
-            <app-enrollee-property title="Postal Code">
-              {{ site.physicalAddress?.postal | postal | default }}
+            <app-enrollee-property title="Job Title"
+                                   [makeBold]="true">
+              {{ site.jobTitle | default }}
             </app-enrollee-property>
 
+            <app-enrollee-property title="Site Address"
+                                   [makeBold]="true">
+              <!-- <div class="mb-3">{{ site.physicalAddress?.street | default }}</div> -->
```

Comment:
Did you mean to leave this in?

Reply:
This line was already there. In my opinion, it is not used but not sure if someone was keeping it on purpose. Can remove.